pkg/schedule/simple_schedule_test.go

```diff
@@ -1,113 +0,0 @@
-// +build legacy
-
-package schedule
-
-import (
-	"testing"
-	"time"
-
-	. "github.com/smartystreets/goconvey/convey"
-)
-
```

Comment:
**To reviewers**:
All test cases from this file have been moved to small and medium tests for windowed schedule
You can find:
 - "test Wait()" nad "invalid interval" in [windowed_schedule_small_test.go](https://github.com/intelsdi-x/snap/pull/1548/files#diff-4c876ad90b686a38e10efef6096ba191R70)
-  tests corresponding to "Simple schedule with no misses" and "with a few misses  you can find [windowed_schedule_medium_test.go](https://github.com/intelsdi-x/snap/pull/1548/files#diff-1b8090dae9b2079edff17c74a4ba55acR36)